pom.xml

```diff
@@ -149,6 +149,12 @@
 			<artifactId>spring-data-elasticsearch</artifactId>
 			<version>1.3.4.RELEASE</version>
 		</dependency>
+		<dependency>
```

Comment:
I don't see any imports of the classes / interfaces from this package. Do we need it?

Reply:
The following error is thrown if we run integration test without this dependency : - 
` java.util.MissingResourceException: Can't find bundle for base name javax.servlet.LocalStrings, locale en_IN`